clang/lib/CodeGen/CGClass.cpp

```diff
@@ -2817,7 +2817,9 @@ void CodeGenFunction::EmitVTablePtrCheckForCall(const CXXRecordDecl *RD,
     RD = LeastDerivedClassWithSameLayout(RD);
 
   auto [Ordinal, _] = SanitizerInfoFromCFICheckKind(TCK);
-  ApplyDebugLocation ApplyTrapDI(*this, SanitizerAnnotateDebugInfo(Ordinal));
+  ApplyDebugLocation ApplyTrapDI(
```

Comment:
I think we should use SanitizerScope here as well

Reply:
Change reverted because it is already in SanitizerScope in some codepaths

Reply:
If so, I maybe make SanitizerScope ++/-- so we can nest them?

Reply:
Probably in follow up patches